src/index.js

```diff
@@ -1,8 +1,9 @@
 import React, { Component } from 'react';
 import csjs from 'csjs';
+import scopedName from 'csjs/lib/scoped-name';
```

Comment:
We could just use the encoder directly if you like, it's nothing more than this: https://github.com/rtsao/csjs/blob/master/lib/scoped-name.js

As i wrote above, `DecoratedComponent.name` gets squashed in prod. All named functions turn annonymous, i tried that first since we already store displayname.